CHANGELOG.md

```diff
@@ -196,6 +259,39 @@
 * fix clang-tidy complain [#14029](https://github.com/mixxxdj/mixxx/pull/14029)
 * ci(dependabot): Open PRs against 2.5 branch instead of main [#14060](https://github.com/mixxxdj/mixxx/pull/14060)
 * Happy New Year 2025! [#14098](https://github.com/mixxxdj/mixxx/pull/14098)
+* Fix warning in Auto DJ test  [#14102](https://github.com/mixxxdj/mixxx/pull/14102)
+* fix: Add `QT_VERSION_CHECK`ed `QCheckBox::checkStateChanged` handlers [#14104](https://github.com/mixxxdj/mixxx/pull/14104)
+* Remove warning introduced in [#13339](https://github.com/mixxxdj/mixxx/pull/13339) [#14109](https://github.com/mixxxdj/mixxx/pull/14109)
+* Fix wrong access to ENV var MIXXX_VCPKG_ROOT instead of CMake setting MIXXX_VCPKG_ROOT [#14146](https://github.com/mixxxdj/mixxx/pull/14146)
+* WOverview: remove unused coefficients [#14145](https://github.com/mixxxdj/mixxx/pull/14145)
+* Fix missing initialization in Rotary() [#14176](https://github.com/mixxxdj/mixxx/pull/14176)
+* (cleanup) remove obsolete focusOutEvent() from WPushButton [#14177](https://github.com/mixxxdj/mixxx/pull/14177)
+* Fix memory leak and use parented_ptr in WTrackMenu [#14199](https://github.com/mixxxdj/mixxx/pull/14199)
+* Fix Clazy warning in main [#14241](https://github.com/mixxxdj/mixxx/pull/14241)
+* Quick fix for [#14197](https://github.com/mixxxdj/mixxx/pull/14197) (`TypeError` in `midi-components-0.0.js`) [#14203](https://github.com/mixxxdj/mixxx/pull/14203)
+* Quick fix for [#14197](https://github.com/mixxxdj/mixxx/pull/14197) (`TypeError` in `midi-components-0.0.js`) [#14251](https://github.com/mixxxdj/mixxx/pull/14251)
```

Comment:
These are the same changes, but different PRs for 2.5 and Main

Reply:
Now both are removed...

Reply:
My idea was that the change will be listed in the 2.5 region, therefore we can remove it here.